.drone.yml

```diff
@@ -0,0 +1,533 @@
+---
```

Comment:
why a jsonnet *and* a yml file?

Reply:
I'm using the jsonnet to generate the .yml
Since I need to do references that would be rather difficult manually

Plus the .yml is signed and I can do some things in drone with that
https://github.com/drGrove/mtls-server/pull/45/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R530